src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -41,6 +41,27 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
                 return false;
             }
 
+            // Default the trigger binding name if a name hasn't
+            // been specified
+            // TODO: Remove this logic and always require it to be explicitly
+            // specified?
+            foreach (var binding in functionMetadata.Bindings.Where(p => p.IsTrigger))
+            {
+                string triggerParameterName = binding.Name;
+                bool triggerNameSpecified = true;
+                if (string.IsNullOrEmpty(triggerParameterName))
+                {
+                    // default the name to simply 'input'
+                    binding.Name = triggerParameterName = "input";
```

Comment:
I can add a central constant for "input". However, as the TODO I put on this code indicates, I wonder if we shouldn't just require the name to be specified. Might change that eventually...